bean-validation/2.0/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Jakarta Bean Validation Spec 2.0"
+date: 2019-08-02
+summary: "Initial Jakarta release of Jakarta Bean Validation"
+---
+Jakarta Bean Validation defines a metadata model and API for JavaBean and method validation.
+
+* [Jakarta Bean Validation 2.0 Specification Document](./bean-validation_2.0.pdf) (PDF)
+* [Jakarta Bean Validation 2.0 Specification Document](./bean-validation_2.0.html) (HTML)
+* [Jakarta Bean Validation 2.0 Javadoc](./apidocs)
+* [Jakarta Bean Validation 2.0 TCK](http://downloads.eclipse.org/jakarta/bean-validation/2.0.5/beanvalidation-tck-dist-2.0.5.zip)
```

Comment:
The URL should be:
http://downloads.eclipse.org/jakarta/bean-validation/2.0/beanvalidation-tck-dist-2.0.5.tar.gz
(unless this is converted to a zip file)
Note directory name is just "2.0".

Reply:
Both zip and tar.gz files exist in the downloads area. The PR has been updated to use the zip version.

Reply:
Is `downloads.eclipse.org` eclipse-internal or geo-restricted service? I get `NXDOMAIN` from DNS.

Reply:
Ok, I'm confused :) So atm. the file is here: http://download.eclipse.org/ee4j/bean-validation/beanvalidation-tck-dist-2.0.5.zip (i.e. no version info in the path, only in the file name)

Is this the correct location? If so, I'll update the PR to link there.

Reply:
Updated as per that URL.

Reply:
The URL in the _index.md file needs to be the URL **where the TCK will be published when it is final**.
Obviously the TCK hasn't been published there yet.

And my URL above was wrong, it should be "download", not "downloads":
http://download.eclipse.org/jakarta/bean-validation/2.0/beanvalidation-tck-dist-2.0.5.zip

Reply:
Ok, updated as per @bshannon's last comment.